script.js

```diff
@@ -134,11 +134,30 @@ const urlconsctructor = function(){
         checkedline.push(i.value)
     }
     Array.prototype.map.call(checkedline, s => s).toString();       
```

Comment:
This line seems not to be needed: I see no effect of the `map`, just the array is converted to string but the value is not assigned to the variable - however, the same is done implicitly on the line below.

Reply:
Just a very detail: when converting the array, `toString()` should be preferred over `toLocaleString()` where the separator depends on locale - the default comma is always needed.

Reply:
> Just a very detail: when converting the array, `toString()` should be preferred over `toLocaleString()` where the separator depends on locale - the default comma is always needed.
it was fixed in last commit from 31.01

Reply:
I still see `toLocaleString()` when URL is created.